.eslintrc.cjs

```diff
@@ -11,7 +11,7 @@ module.exports = {
     'plugin:svelte/recommended',
     'prettier',
   ],
-  plugins: ['svelte', 'simple-import-sort'],
+  plugins: ['svelte', 'simple-import-sort', 'json-files'],
```

Comment:
Without any rules and an overrides section mentioning `.json`, I don't think adding this plugin does anything

Reply:
It does! Because in this case the list of files is passed explicitly to eslint (versus being generated off the include and exclude lists).

Reply:
Still, we're not configuring any rules. Is this just here so a JSON parser is used and ESLint doesn't choke when a JSON file comes in the diff list? Would adding JSON to the ESLint ignore achieve the same thing?

Reply:
I could add json files to the ignore list, but if I can get eslint to check them instead, hey, why not?